src/Document.php

```diff
@@ -1714,11 +1714,21 @@ public static function dropdown($options = [])
             $values[$data['id']] = $data['name'];
         }
         $rand = mt_rand();
-        $out  = Dropdown::showFromArray('_rubdoc', $values, ['width'               => '30%',
+        $readonly = $p['readonly'] ?? false;
```

Comment:
Please add a `$p['readonly'] = false` on line 1673. Having all the default options values at the same place is easier to read.

```suggestion
        $readonly = $p['readonly'];
```